hoomd/md/TwoStepNPTMTK.h

```diff
@@ -255,6 +245,9 @@ class PYBIND11_EXPORT TwoStepNPTMTK : public IntegrationMethodTwoStep
 
     Scalar m_gamma; //!< Optional damping factor for box degrees of freedom
 
+    std::vector<Scalar> m_thermostat; //!< thermostat degrees of freedom
+    std::vector<Scalar> m_barostat;   //!< barostat degrees of freedom
```

Comment:
Since the sizes of these arrays are static, why not declare them as `std::array` instead of `std::vector`? It would make their intended usage clearer to someone trying to understand the code.

Reply:
No reason, that is a good idea.

Reply:
I suggested offline to use `m_xi`, `m_eta`, etc... in place of the thermostat array to avoid the potential for error when extracting values from the array by index.

Reply:
That's a good idea, could also just document which index corresponds to which variable in a comment above where the member variable is declared.

Reply:
Oh, @joaander I misunderstood your suggestion. I can switch to that model then. Although I may create simple structs to serve that purpose then.